students/Sean_Tasaki/Lesson2/series.py

```diff
@@ -0,0 +1,83 @@
+def fibonacci(n):
+    """Return the nth value in the fibonacci sequence (starting with zero index.
+    """
+    if n == 1:
+        return 1
+    if n == 0:
+        return 0
+    if n < 0:
+        print("Invalid data")
+    else:
+    	return fibonacci(n - 2) + fibonacci(n - 1)
+
+def lucas(n):
+	"""Return the nth value in the Lucas numbers series (starting with zero index.
```

Comment:
Looks like the indentation level switches from 4 spaces to tabs here. Be sure to always use 4 spaces for indentation, you can configure your editor to enter four spaces automatically when you hit the tab key